packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.less

```diff
@@ -0,0 +1,19 @@
+@import (reference) "@nova-ui/bits/sdk/less/nui-framework-variables.less";
+@import (reference) "@nova-ui/bits/sdk/less/mixins.less";
+
+.legend-container {
```

Comment:
I think we can use the existing `.nui-legend-series` class here instead. This class is automatically applied to the nui-legend-series element.